digits/dataset/images/classification/views.py

```diff
@@ -156,12 +156,15 @@ def from_files(job, form):
     """
     # labels
     if form.textfile_use_local_files.data:
-        job.labels_file = form.textfile_local_labels_file.data.strip()
+        labels_file_from = form.textfile_local_labels_file.data.strip()
+        labels_file_to = os.path.join(job.dir(), utils.constants.LABELS_FILE)
+        import shutil
```

Comment:
maybe you can move this import to the top (line 5), where the other standard import packages are